Kernel/KBufferBuilder.cpp

```diff
@@ -51,7 +51,7 @@ OwnPtr<KBuffer> KBufferBuilder::build()
 
 ErrorOr<KBufferBuilder> KBufferBuilder::try_create()
 {
-    auto buffer = TRY(KBuffer::try_create_with_size(4 * MiB, Memory::Region::Access::ReadWrite));
+    auto buffer = TRY(KBuffer::try_create_with_size("KBufferBuilder"sv, 4 * MiB, Memory::Region::Access::ReadWrite));
```

Comment:
Not your fault, but 4 MiB???

Reply:
Let's hope not too many of those live at the same time. :^)